.vscode/tasks.json

```diff
@@ -251,6 +251,21 @@
             "type": "shell",
             "command": "dotnet tool install -g dotnet-certificate-tool -g && certificate-tool add --file ${workspaceFolder}/dev/dev.pfx --password '${input:certPassword}'",
             "problemMatcher": []
+        },
+        {
+            "label": "buildMsSqlMigratorUtility",
+            "hide": true,
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/util/MsSqlMigratorUtility/MsSqlMigratorUtility.csproj"
+            ],
+            "problemMatcher": "$msCompile",
+            "group": {
+                "kind": "build",
+                "isDefault": true
```

Comment:
Not _quite_ sure what this would do, but this doesn't make sense as a default task, to me